elibrary-core/source/src/controller/java/com/javafee/elibrary/core/settingsform/PasswordChangePanelEvent.java

```diff
@@ -1,9 +1,11 @@
 package com.javafee.elibrary.core.settingsform;
 
 import java.text.MessageFormat;
+import java.util.Objects;
```

Comment:
Are those two imports the only change in the file? 
If yes, then please remove it. 🗑️

Reply:
This is done